docs/foundations/typography/usage/type-usage-best-practice-3-do.svg

```diff
@@ -3,6 +3,7 @@
         fill="#0066CC" />
   <g clip-path="url(#clip0_623_5192)">
```

Comment:
it's kinda annoying, but we have to sanitize and namespace all the IDs, now that these are inlined